tests/test_ag3.py

```diff
@@ -664,6 +664,29 @@ def test_snp_allele_frequencies():
     assert np.any(df.max_af == 0)
 
 
+def test_snp_allele_frequencies_0_cohort():
+    ag3 = setup_ag3()
+    cohorts = {
+        "bf_2050_col": "country == 'Burkina Faso' and year == 2050 and species == 'coluzzii'",
+    }
+
+    try:
+        _ = ag3.snp_allele_frequencies(
+            transcript="AGAP009194-RA",
+            cohorts=cohorts,
+            site_mask="gamb_colu",
+            sample_sets="v3_wild",
+            drop_invariant=True,
+        )
+    except ValueError:
+        # The exception was raised as expected
+        pass
+    else:
+        # If we get here, then the ValueError was not raised
+        # raise an exception so that the test fails
+        raise AssertionError("ValueError was not raised")
```

Comment:
The conventional way to test whether an exception is raised is to use [pytest.raises](https://docs.pytest.org/en/6.2.x/reference.html#pytest.raises). E.g.:

```suggestion
    with pytest.raises(ValueError):
        _ = ag3.snp_allele_frequencies(
            transcript="AGAP009194-RA",
            cohorts=cohorts,
            site_mask="gamb_colu",
            sample_sets="v3_wild",
            drop_invariant=True,
        )
```